readthedocsext/theme/templates/profiles/private/migrate_to_gh_app.html

```diff
@@ -0,0 +1,303 @@
+                  This new app offers more granular permissions and better GitHub integration.
+                  Learn more in our <a href="{{ blog_post }}">blog post</a>.
+                {% endblocktrans %}
+              </p>
+
+              {% if has_multiple_github_accounts %}
+                <div class="ui message warning">
+                  <i class="fad fa-warning icon"></i>
+                  {% url 'socialaccount_connections' as socialaccount_connections %}
+                  {% blocktrans trimmed with socialaccount_connections=socialaccount_connections github_account=old_github_account.extra_data.login %}
+                    You have <a href="{{ socialaccount_connections }}">multiple GitHub accounts</a> linked,
+                    <strong>you'll need to repeat the migration for each account</strong>.
+
+                    The current account to migrate is <a href="https://github.com/{{ github_account }}" target="_blank">{{ github_account }}</a>;
+                    you can skip repositories this account doesn't have access to.
+                  {% endblocktrans %}
+                </div>
+              {% endif %}
+
+              <p>
+                {% blocktrans %}
+                This guide will help you migrate your account to the new GitHub App.
+                Some things you should know:
+                {% endblocktrans %}
+              </p>
+
+              <p>
+                <ul>
+                  <li>
+                    {% blocktrans %}
+                    Migrating all projects is optional,
+                    you can skip the &quot;Install GitHub App&quot; and &quot;Migrate Projects&quot; steps.
+                    {% endblocktrans %}
+                  </li>
+                  <li>
+                    {% blocktrans %}
+                    Connecting your account to the new GitHub App and revoking the old GitHub OAuth app is required,
+                    as the old app is deprecated and will be removed.
+                    {% endblocktrans %}
+                  </li>
+                  <li>
+                    {% blocktrans %}
+                    Some steps require you to do changes on GitHub,
+                    refresh the page after making the changes to see updates.
+                    {% endblocktrans %}
+                  </li>
+                  <li>
+                    {% blocktrans %}
+                    New projects imported after connecting your account to the new GitHub App don't need migration,
+                    as they will be automatically connected to the new GitHub App.
+                    {% endblocktrans %}
+                  </li>
+                  <li>
+                    {% blocktrans trimmed with manual_migration_docs="#" %}
+                      After revoking the old GitHub OAuth app, you won't be able to auto-migrate projects.
+                      <a href="{{ manual_migration_docs }}">Manual migration</a> is always an option.
```

Comment:
This isn't a great place to push the user towards documentation as we don't want to distract the user from this UI. This note is adding to information overload on this page, and the view is already very text dense -- and the more text the less likely all of it will be read.

This seems more like a note for edge cases. If this is not necessary for initial migration, I think we can cut this and make this proper UI in the project admin.

```suggestion
                      After revoking our old GitHub OAuth app, you won't be able to automatically migrate projects.
                      <a href="{{ manual_migration_docs }}">Manual migration</a> is always an option.
```